package.json

```diff
@@ -27,23 +28,30 @@
     "@tsndr/cloudflare-worker-jwt": "^1.1.4",
     "add": "^2.0.6",
     "classnames": "^2.3.1",
-    "next": "^12.1.6",
+    "next": "^12.3.0",
     "next-seo": "^4.24.0",
-    "react": "^18.1.0",
-    "react-dom": "^18.1.0",
-    "react-icons": "^4.2.0"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-icons": "^4.2.0",
+    "uuid": "^8.3.2"
   },
   "devDependencies": {
+    "@babel/core": "^7.18.10",
+    "@babel/preset-env": "^7.18.10",
+    "@babel/preset-typescript": "^7.18.6",
     "@next/eslint-plugin-next": "^11.1.0",
     "@tailwindcss/aspect-ratio": "^0.2.0",
     "@tailwindcss/forms": "^0.3.2",
     "@tailwindcss/typography": "^0.4.0",
+    "@types/jest": "^28.1.6",
     "@types/node": "^14.14.43",
     "@types/react": "^17.0.4",
+    "@types/uuid": "^8.3.4",
```

Comment:
not sure if this was part of the `uuid` package install, but if it was, pop this out too if ya need to